lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +217,49 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
```

Comment:
without initialized tagconverter you will not be able to process pos tags requried for 6